meson.build

```diff
@@ -641,7 +641,11 @@ foreach prog : progs
                                    '/usr/sbin/' + prog[0],
                                    '/sbin/' + prog[0],
                                    required: false)
-                path = exe.found() ? exe.path() : prog[1]
+                if meson.version().version_compare('>= 0.55')
+                        path = exe.found() ? exe.full_path() : prog[1]
+                else
+                        path = exe.found() ? exe.path() : prog[1]
+                endif
```

Comment:
> There's no reason to bump the minimum version of Meson for such a trifling change.

Agreed. Thanks! I was thinking more about `install_tag` when I was writing that.

Reply:
install_tag is different, yeah.

Specifically, because I'm the current state of affairs it prevents a warning-free build (that is, it's *impossible* no matter what options or configurations you use, to suppress the warning).

Reply:
@eli-schwartz Thank you for the suggestion. The change is dropped.